docs/en/pipeline-mgmt/basic.md

```diff
@@ -0,0 +1,26 @@
+# Basic configuration
+
+For the clickstream project, you can specify the basic configuration of the data pipeline and set the following configurations:
+
+* **AWS Region**: select the region where the pipeline is located. If you specify the region where AWS services are not available, the feature will be disabled by default. Check [the region table][region-table] for feature availability.
+* **VPC**: specify the VPC where the compute resources of the pipeline will be located. The VPC needs to meet the below criteria for running the pipeline workload.
+    {%
+      include-markdown "./vpc-prerequisites.md"
+    %}
+* **Data collection SDK**: specify the SDK type that the client uses.
+    - If you choose **Clickstream SDK**, you can check the [SDK manual][clickstream-sdks] for available Clickstream SDKs.
```

Comment:
If you choose **Clickstream SDK**, please refer to [SDK manual][clickstream-sdks] for the available Clickstream SDKs and integration guides.